packages/app/src/assets/DataIcon.js

```diff
@@ -1,74 +1,119 @@
             >
-                <g id="Icon/12x12/Data">
-                    <g id="icon_data_small">
-                        <rect id="bounds" x="0" y="0" width="12" height="12" />
-                        <rect
-                            id="Rectangle-18"
-                            stroke="#494949"
-                            x="0.5"
-                            y="0.5"
-                            width="4"
-                            height="4"
-                        />
-                        <rect
-                            id="Rectangle-18"
-                            stroke="#494949"
-                            x="7.5"
-                            y="0.5"
-                            width="4"
-                            height="4"
-                        />
-                        <rect
-                            id="Rectangle-18"
-                            stroke="#494949"
-                            x="7.5"
-                            y="7.5"
-                            width="4"
-                            height="4"
-                        />
-                        <rect
-                            id="Rectangle-18"
-                            stroke="#494949"
-                            x="0.5"
-                            y="7.5"
-                            width="4"
-                            height="4"
-                        />
+                <g id="Artboard" transform="translate(-7.000000, -9.000000)">
+                    <g
+                        id="icon_data_new"
+                        transform="translate(7.000000, 9.000000)"
+                    >
+                        <rect id="frame" x="0" y="0" width="16" height="16" />
+                        <g
+                            id="database-2"
+                            transform="translate(2.000000, 2.000000)"
+                            stroke="#000000"
+                            strokeLinecap="round"
+                            strokeLinejoin="round"
+                        >
+                            <ellipse
+                                id="Oval"
+                                cx="5.5"
+                                cy="2.5"
+                                rx="5.5"
+                                ry="2.5"
+                            />
+                            <path
+                                d="M0,3 L0,5.59978665 C0,6.9252378 2.46253333,8 5.5,8 C8.53746667,8 11,6.92577118 11,5.59978665 L11,3"
+                                id="Shape"
+                            />
+                            <path
+                                d="M0,5 L0,8.6 C0,9.92533333 2.46253333,11 5.5,11 C8.53746667,11 11,9.92533333 11,8.6 L11,5"
+                                id="Shape"
+                            />
+                        </g>
                     </g>
                 </g>
             </g>
         </svg>
     );
 };
 
+// const DataIcon = ({ style }) => {
```

Comment:
Probably should just remove commented out lines, but that can also be done later